### PR TITLE
Shelving

### PR DESCRIPTION
**Pull request by danich1**
Quick PR to add the shelving power to our repo.

### CODE REVIEW

**Comment by dhimmel** [comment]
Let's merge https://github.com/greenelab/pubtator/pull/10 before this PR. You should be able to rebase... I don't see any conflicts.

File: scripts/shelve_tags.py [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+    """ Stores annotations into python's shelve module (Dictionary)
+
+    Keywords:
+    input_file - the name of the element tagged file
+    output_file - the name of the file to be written out
+    """
+    with open(input_file, 'r') as g:
+        d = shelve.open(output_file)
+        reader = csv.DictReader(g, delimiter='\t')
+        temp = []
```
Comment (dhimmel) [inline scripts/shelve_tags.py]: `itertools.groupby` will make things much easier / clearer.

**Comment by dhimmel** [comment]
@danich1 can you remind me what the shelve is storing? What are the keys, what are the values?

I'm thinking it may make sense to have @zietzm finish this PR, since he's already familiar with this repo. Just I can't remember exactly what we need in the shelve.

**Comment by danich1** [comment]
Sorry for the delay. The shelve is storing document-specific tags that pubtator provided. The values are the tags that correspond to title and abstract and the keys are pubmed ids. 

Example.
  variable[pubmed id] = [[Biological type, MESH id, start, stop]...]
  shelve var[1] = [["Disease", "D016464", "29","54"], ...]

**Comment by danich1** [comment]
Closing this outdated pull request. Will reopen another time.

**Pull request closed without merging**